datasets/sentinel-products-ca-mirror.yaml

```diff
@@ -0,0 +1,30 @@
+Name: Sentinel Products Canada Mirror
+Description: "The official Government of Canada 🍁 mirror for Sentinel-1,2,3 data under the EU's Copernicus programme focused on Canada-wide coverage. Additional Sentinel-2 and Sentinel-6 products over Canada are retrieved from [EUMETSAT](https://eoportal.eumetsat.int/cas/login)"
+Documentation: https://roda.sentinel-hub.com/sentinel-s1-l1c/GRD/readme.html
```

Comment:
This documentation link is specific to Sentinel-1 and refers to the GRD datasets in SentinelHub. Do you have a link that references the datasets that you plan to provide for this listing?

Reply:
fixed